requirements.txt

```
@@ -1,4 +1,4 @@
-EGCG-Core==0.11
+EGCG-Core==0.11.1
```

That should be the newer version

```suggestion
EGCG-Core==0.11.2
```